Brittany Crocker/Week 01/Calculator/index.html

```diff
@@ -0,0 +1,12 @@
+<html>
+  <head>
+    <script src="calculator-part-01.js"></script>
+
+<meta charset="utf-8">
+    </script>
+    <title>Javascript Exercises</title>
+  </head>
+  <body>
+  <p>Please check the Javascript console for program output.</p>
+  </body>
+</html>
```

Comment:
This seems  a little all over the place... should be more like this: 
```
<html>
  <head>
    <script src="calculator-part01.js"></script>
    <meta charset="utf-8">
    <title>Javascript Exercises</title>
  </head>
  <body>
    <p>Please check the Javascript console for program output.</p>
  </body>
</html>
```
Also keep in mind the indentation of your code.